### DISCUSSION TITLE
Call to undefined method Spatie\MediaLibrary\Models\Media::addMediaConversion()

### DISCUSSION THREAD

**Discussion by jonjieviduya** [spatie/laravel-medialibrary#2048] [Q&A] [answered]
I just followed the [documentation](https://spatie.be/docs/laravel-medialibrary/v7/converting-images/defining-conversions) says "**Internally, spatie/image is used to manipulate the images. You can use any manipulation function from that package.**" so I thought I can use the `addMediaConversion` method. Any idea with this?

**Comment by freekmurze** [comment, marked as answer]
You can see a few examples here: Targeted the upgrade-laravel-8 branch. When that one gets merged, GitHub will automatically retarget this PR to the master branch.
